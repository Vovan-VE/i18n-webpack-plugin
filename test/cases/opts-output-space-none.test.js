import fs from 'fs';
import processFile from '../cases.setup';

describe('opts-output-space-none', () => {
  const extracted = [];
  const outputSpace = null;

  beforeAll(() => processFile('opts-output-space-none.code.js', { outputSpace })
    .then(({ files }) => {
      Object.keys(files).forEach((name) => {
        extracted.push({
          name,
          content: fs.readFileSync(files[name], 'utf8'),
        });
      });
    }));

  it('should return extracted keys', () => {
    expect(extracted).toMatchSnapshot();
  });
});
