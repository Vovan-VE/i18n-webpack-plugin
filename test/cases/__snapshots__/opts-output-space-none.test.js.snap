// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`opts-output-space-none should return extracted keys 1`] = `
Array [
  Object {
    "content": "{\\"category1\\":{\\"Message 1\\":\\"\\",\\"Message 2\\":\\"\\"},\\"category2\\":{\\"Message 3\\":\\"\\"}}
",
    "name": "opts-output-space-none@en-US",
  },
]
`;
